### PR TITLE
chore(ecs): add additional missing unit tests

### PR DESCRIPTION
**Pull request by iamhopaul123**
Add missing unit tests for aws-ecs.

Please read the [contribution guidelines](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md) and follow the pull-request checklist.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by iamhopaul123** [comment]
@pkandasamy91 please review

File: packages/@aws-cdk/aws-ecs/test/test.container-definition.ts [hunk 1]
```diff
@@ -202,6 +403,28 @@ export = {
         test.equal(actual, expected);
         test.done();
       },
+
+      "throws when calling ingressPort with no PortMappings"(test: Test) {
```
[the remark this answers is not in the document]
Comment (iamhopaul123) [reply packages/@aws-cdk/aws-ecs/test/test.container-definition.ts]: yeah I think one would be enough according to the logic.

File: packages/@aws-cdk/aws-ecs/test/ec2/test.ec2-service.ts [hunk 2]
```diff
@@ -995,15 +991,179 @@ export = {
...
+      }, /Cannot use a Classic Load Balancer if NetworkMode is None. Use Host or AwsVpc instead./);
+
+      test.done();
+    }
+  },
+
+  "attachToApplicationTargetGroup": {
+    "allows network mode of task definition to be other than none"(test: Test) {
+      // GIVEN
+    const stack = new cdk.Stack();
```
Comment (pkandasamy91) [inline packages/@aws-cdk/aws-ecs/test/ec2/test.ec2-service.ts]: nitpick: indentation

File: packages/@aws-cdk/aws-ecs/test/fargate/test.fargate-service.ts [hunk 3]
```diff
@@ -178,6 +178,29 @@ export = {
       test.done();
     },
 
+    "throws when task definition is not EC2 compatible"(test: Test) {
```
Comment (pkandasamy91) [inline packages/@aws-cdk/aws-ecs/test/fargate/test.fargate-service.ts]: This should be 'throws when task definition is not Fargate compatible

**Pull request merged**